COLLIDER_EVENTS.md

```diff
@@ -0,0 +1,66 @@
+# Overview
+
+Collider events from the client are broken down in to two types: Collision and Trigger.  Each of these event types are raised for one of two event states: 'enter' or 'exit'.
+
+# Collision Event
+
+The collision event is raised when two actors that both have colliders on them participate in a collision event, one for when the collision enters and one for when the collision exits..  This collision event will contains data that is passed to the event handlers attached for the collision event and state.
+
+## Collision Data
+
+Collision data is specific to only collision events and contains information on the other actor, as well as physics related information about the collision.
+
+```ts
+export interface ContactPoint {
+    normal: Vector3;
+    point: Vector3;
+    separation: number;
+}
+
+export interface CollisionData {
+    actorHit?: Actor;
+    contacts: ContactPoint[];
+    impulse: Vector3;
+    relativeVelocity: Vector3;
+}
+```
+
+# Trigger Volume Events
+
+A trigger volume event is raised when an actor that has a collider on it enters or exits the bounds of another actor's collider that is configured to be a trigger. This trigger event will contain data that is passed to the event containining the actor id of the other actor.
```

Comment:
> enters or exits

So triggers are not fired continuously while within the bounds, correct?

Reply:
Right.  Just enter and exit.